src/admin-views/settings/tickets-commerce/stripe/connect/active/paypal-status.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * The Template for displaying the Tickets Commerce PayPal status.
+ *
+ * @version 5.1.10
+ *
+ * @since   5.1.10
+ *
+ * @var Tribe__Tickets__Admin__Views                  $this                  [Global] Template object.
+ * @var string                                        $plugin_url            [Global] The plugin URL.
+ * @var TEC\Tickets\Commerce\Gateways\PayPal\Merchant $merchant              [Global] The merchant class.
+ * @var TEC\Tickets\Commerce\Gateways\PayPal\Signup   $signup                [Global] The Signup class.
```

Comment:
```suggestion
 * The Template for displaying the Tickets Commerce Stripe status.
 *
 * @version TBD
 *
 * @since   TBD
 *
 * @var Tribe__Tickets__Admin__Views                  $this                  [Global] Template object.
 * @var string                                        $plugin_url            [Global] The plugin URL.
 * @var TEC\Tickets\Commerce\Gateways\Stripe\Merchant $merchant              [Global] The merchant class.
 * @var TEC\Tickets\Commerce\Gateways\Stripe\Signup   $signup                [Global] The Signup class.
```